### PR TITLE
Add changelog for what will become 2.0.2

### PR DESCRIPTION
**Pull request by ashb**
Preparing for 2.0.2 release process, here is the changelog of user-facing, non-provider changes

### CODE REVIEW

**Comment by ashb** [comment]
I'm going to add one or two more commits

File: CHANGELOG.txt [hunk 1]
```diff
@@ -1,5 +1,98 @@
+Airflow 2.0.2, 2021-04-19
+-------------------------
+
+Bug Fixes
+"""""""""
+
+* Fix missing on_load trigger for folder-based plugins (#15208)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
* Fix missing ``on_load`` trigger for folder-based plugins (#15208)
```

File: CHANGELOG.txt [hunk 2]
```diff
@@ -1,5 +1,98 @@
...
+* ``kubernetes cleanup-pods`` subcommand will only clean up Airflow-created Pods (#15204)
+* Fix password masking in CLI action_logging (#15143)
```
Comment (kaxil) [inline CHANGELOG.txt:10]:
```suggestion
* Fix password masking in CLI ``action_logging`` (#15143)
```

File: CHANGELOG.txt [hunk 3]
```diff
@@ -1,5 +1,98 @@
...
+* Fix url generation for TriggerDagRunOperatorLink (#14990)
```
Comment (kaxil) [inline CHANGELOG.txt:11]:
```suggestion
* Fix url generation for ``TriggerDagRunOperatorLink`` (#14990)
```

File: CHANGELOG.txt [hunk 4]
```diff
@@ -1,5 +1,98 @@
...
+* Restore base lineage backend (#14146)
+* Unable to trigger backfill or manual jobs with Kubernetes executor. (#14160)
+* Bugfix: Task docs are not shown in the Task Instance Detail View (#15191)
+* Bugfix: Fix overriding `pod_template_file` in KubernetesExecutor (#15197)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
* Bugfix: Fix overriding ``pod_template_file`` in KubernetesExecutor (#15197)
```

File: CHANGELOG.txt [hunk 5]
```diff
@@ -1,5 +1,98 @@
...
+* Bugfix: resources in `executor_config` breaks Graph View in UI (#15199)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
* Bugfix: resources in ``executor_config`` breaks Graph View in UI (#15199)
```

File: CHANGELOG.txt [hunk 6]
```diff
@@ -1,5 +1,98 @@
...
+* Fix celery executor bug trying to call len on map (#14883)
+* Fix bug in airflow.stats timing that broke dogstatsd mode (#15132)
```
Comment (kaxil) [inline CHANGELOG.txt:18]:
```suggestion
* Fix bug in ``airflow.stats`` timing that broke ``dogstatsd`` mode (#15132)
```

File: CHANGELOG.txt [hunk 7]
```diff
@@ -1,5 +1,98 @@
...
+* Avoid scheduler/parser manager deadlock by using non-blocking IO (#15112)
+* Re-introduce dagrun.schedule_delay metric (#15105)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
* Re-introduce ``dagrun.schedule_delay`` metric (#15105)
```

File: CHANGELOG.txt [hunk 8]
```diff
@@ -1,5 +1,98 @@
...
+* Compare string values, not if strings are the same object in Kube executor(#14942)
+* Pass queue to BaseExecutor.execute_async like in airflow 1.10 (#14861)
```
Comment (kaxil) [inline CHANGELOG.txt:22]:
```suggestion
* Pass queue to ``BaseExecutor.execute_async`` like in airflow 1.10 (#14861)
```

File: CHANGELOG.txt [hunk 9]
```diff
@@ -1,5 +1,98 @@
...
+* Scheduler: Remove TIs from starved pools from the critical path. (#14476)
+* Remove extra/needless deprecation warnings from airflow.contrib module (#15065)
```
Comment (kaxil) [inline CHANGELOG.txt:24]:
```suggestion
* Remove extra/needless deprecation warnings from ``airflow.contrib`` module (#15065)
```

File: CHANGELOG.txt [hunk 10]
```diff
@@ -1,5 +1,98 @@
...
+* Fix support for long dag_id and task_id in KubernetesExecutor (#14703)
```
Comment (kaxil) [inline CHANGELOG.txt:25]:
```suggestion
* Fix support for long dag_id and task_id in ``KubernetesExecutor`` (#14703)
```

File: CHANGELOG.txt [hunk 11]
```diff
@@ -1,5 +1,98 @@
...
+* Sort lists, sets and tuples in Serialized DAGs (#14909)
+* Simplify cleaning string passed to origin param (#14738) (#14905)
+* Fix error when running tasks with Sentry integration enabled. (#13929)
+* Webserver: Sanitize string passed to origin param (#14738)
+* Fix losing duration < 1 secs in tree (#13537)
+* Pin SQLAlchemy to <1.4 due to breakage of sqlalchemy-utils (#14812)
```
Comment (kaxil) [inline CHANGELOG.txt:31]:
```suggestion
* Pin SQLAlchemy to <1.4 due to breakage of ``sqlalchemy-utils`` (#14812)
```

File: CHANGELOG.txt [hunk 12]
```diff
@@ -1,5 +1,98 @@
...
+* Fix KubernetesExecutor issue with deleted pending pods (#14810)
+* Default to Celery Task model when backend model does not exist (#14612)
+* Bugfix: Plugins endpoint was unauthenticated (#14570)
+* BugFix: fix DAG doc display (especially for TaskFlow DAGs) (#14564)
+* BugFix: TypeError in airflow.kubernetes.pod_launcher's monitor_pod (#14513)
```
Comment (kaxil) [inline CHANGELOG.txt:36]:
```suggestion
* BugFix: TypeError in ``airflow.kubernetes.pod_launcher``'s ``monitor_pod`` (#14513)
```

File: CHANGELOG.txt [hunk 13]
```diff
@@ -1,5 +1,98 @@
...
+* Bugfix: Fix wrong output of tags and owners in dag detail API endpoint (#14490)
+* Fix logging error with task error when JSON logging is enabled (#14456)
+* Fix statsd metrics not sending when using daemon mode (#14454)
+* Gracefully handle missing start_date and end_date for DagRun (#14452)
+* BugFix: Serialize max_retry_delay as a timedelta (#14436)
```
Comment (kaxil) [inline CHANGELOG.txt:41]:
```suggestion
* BugFix: Serialize ``max_retry_delay`` as a timedelta (#14436)
```

File: CHANGELOG.txt [hunk 14]
```diff
@@ -1,5 +1,98 @@
...
+* Fix crash when user clicks on  "Task Instance Details" caused by start_date being None (#14416)
+* BugFix: Fix TaskInstance API call fails if a task is removed from running DAG (#14381)
+* Scheduler should not fail when invalid executor_config is passed (#14323)
```
Comment (kaxil) [inline CHANGELOG.txt]:
```suggestion
* Scheduler should not fail when invalid ``executor_config`` is passed (#14323)
```

File: CHANGELOG.txt [hunk 15]
```diff
@@ -1,5 +1,98 @@
...
+* Fix bug allowing task instances to survive when dagrun_timeout is exceeded (#14321)
```
Comment (kaxil) [inline CHANGELOG.txt:45]:
```suggestion
* Fix bug allowing task instances to survive when ``dagrun_timeout`` is exceeded (#14321)
```

File: CHANGELOG.txt [hunk 16]
```diff
@@ -1,5 +1,98 @@
...
+* Updates 3.6 limits for latest versions of a few libraries (#15209)
+* Adds Blinker dependency which is missing after recent changes (#15182)
+* Remove 'conf' from search_columns in DagRun View (#15099)
+* More proper default value for namespace in K8S cleanup-pods CLI (#15060)
+* Faster default role syncing during webserver start (#15017)
+* Speed up webserver start when there are many DAGs (#14993)
+* Much easier to use and better documented Docker image (#14911)
+* Use ``libyaml`` C library when available. (#14577)
+* Don't create unittest.cfg when not running in unit test mode (#14420)
+* Webserver: Allow Filtering TaskInstances by queued_dttm (#14708)
```
Comment (kaxil) [inline CHANGELOG.txt:67]:
```suggestion
* Webserver: Allow Filtering TaskInstances by ``queued_dttm`` (#14708)
```

File: CHANGELOG.txt [hunk 17]
```diff
@@ -1,5 +1,98 @@
...
+* Docs: Clarify behavior of delete_worker_pods_on_failure (#14958)
+* Create a documentation package for Docker image (#14846)
+* Multiple minor doc (OpenAPI) fixes (#14917)
+* Replace Graph View Screenshot to show Auto-refresh (#14571)
+
+Misc/Internal
+"""""""""""""
+
+* Import Connection lazily in hooks to avoid cycles (#15361)
+* Rename last_scheduler_run into last_parsed_time, and ensure it's updated in DB (#14581)
```
Comment (kaxil) [inline CHANGELOG.txt:92]:
```suggestion
* Rename ``last_scheduler_run`` into ``last_parsed_time``, and ensure it's updated in DB (#14581)
```

**Review by kaxil — COMMENTED** [review]
nit, can you change `* ` to `- ` -- does not matter for rst but just for viewing raw file for consistency

File: CHANGELOG.txt (hunk 4 above)
Comment (ashb) [reply CHANGELOG.txt]: Does KubeExecutor need to be code-ified?

It appears there are many cases already in the changelog where we don't have class names in code blocks.

Comment (kaxil) [reply CHANGELOG.txt]: not strictly needed tbh -- just looks nice :D http://airflow.apache.org/docs/apache-airflow/stable/changelog.html

**Review by kaxil — APPROVED** [review]

**Comment by ashb** [comment]
I've done some of the code-blocks, but not all :)

**Pull request merged**